src/main/kotlin/com/openlattice/codex/MessageRequest.kt

```diff
@@ -6,5 +6,6 @@ data class MessageRequest (
         val organizationId: UUID,
         val messageEntitySetId : UUID,
         val messageContents: String,
-        val phoneNumber: String
+        val phoneNumber: String,
+        var senderId: String? = null
```

Comment:
As you know, I hate having nullable fields unless absolutely necessary. It looks like we only create `MessageRequests` in one place where we can pretty readily set the senderId to some constant indicating that it was sent by us. Is there any reason this has to stay nullable?

Reply:
Okay nvm, I now see that we're actually doing construction implicitly in `CodexController.sendOutgoingText`

Reply:
Yeahhh I kind of hate that I've implemented it this way lol -- that's actually the main reason this is a draft PR right now but I wanted to get something working and then hopefully reduce the ugliness after

Reply:
How about creating 2 classes, one without and a child class with the `senderId`?

Reply:
yeahhh potentially, although I don't love creating a whole new pojo + serializer for something that will only be used once / basically for a single extra string field. but I also don't love the nullable.

Reply:
* poko? 🤔

Reply:
My vote is to require it and set it everywhere though I believe that will require it to be set by something when it gets deserialized as the requestbody